package.json

```
@@ -1,17 +1,17 @@
 {
   "name": "ffc-demo-payment-service",
-  "version": "2.3.1",
+  "version": "2.4.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "build:watch": "nodemon --exec npm run build --ext scss --legacy-watch",
     "migrate": "sequelize db:migrate",
     "pretest": "npm run test:lint",
-    "start:watch": "nodemon --inspect=0.0.0.0 --ext js --legacy-watch index.js",
-    "test": "npm run test:unit",
+    "test": "jest --forceExit --detectOpenHandles",
```

Do we want to remove the --forceExit here
```suggestion
    "test": "jest --detectOpenHandles",
```

---

John's addressing this in his PR